src/coreclr/jit/llvmcodegen.cpp

```diff
@@ -3290,3 +3310,9 @@ llvm::Intrinsic::ID Llvm::getLlvmIntrinsic(NamedIntrinsic intrinsicName) const
             return llvm::Intrinsic::not_intrinsic;
     }
 }
+
+void Llvm::displayValue(Value* value)
+{
+    // TODO-LLVM: support JitStdOutFile here.
+    value->print(llvm::outs());
+}
```

Comment:
It turns out that `value->dump()` prints to `stderr` by default, which breaks the common pattern of redirecting output to a file.